src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        Colors color = colors[random.nextInt(5)];
+        return color.toString();
```

Comment:
better use `.name()`
```suggestion
        return color.name();
```

Reply:
not fixed